hwy/abort.cc

```diff
@@ -20,6 +20,10 @@
 namespace hwy {
 
 namespace {
+
+#ifdef HWY_HEADER_ONLY
+inline
```

Comment:
I think for header files, we want static inline just to be sure/safe.

Reply:
Doesn't anonymous namespace take the place of static in this case ?

Reply:
Probably a good idea, but I don't think that's issue with the compile error. Compile currently chokes on
`mask_slide_test.cc`. so it is able to compile a number of other tests successfully. When you have a chance, take a look as I can't make head or tail of the error.

Reply:
Interesting. What's the error? I'm not seeing this in the build logs, currently they are failing at CMake already due to the CXX project setting.

Reply:
Only one test fails, with these kinds of errors:

```
/home//src/highway/hwy/tests/mask_mem_test.cc:297:7: error: redefinition of ‘class hwy::N_SCALAR::TestStoreMaskBits’
  297 | class TestStoreMaskBits {
      |       ^~~~~~~~~~~~~~~~~
In file included from /home//src/highway/hwy/foreach_target.h:327,
                 from /home//src/highway/hwy/tests/mask_mem_test.cc:27:
/home//src/highway/hwy/tests/mask_mem_test.cc:297:7: note: previous definition of ‘class hwy::N_SCALAR::TestStoreMaskBits’
  297 | class TestStoreMaskBits {
      |       ^~~~~~~~~~~~~~~~~
/home//src/highway/hwy/tests/mask_mem_test.cc:377:19: error: redefinition of ‘void hwy::N_SCALAR::TestAllStoreMaskBits()’
  377 | HWY_NOINLINE void TestAllStoreMaskBits() {
      |                   ^~~~~~~~~~~~~~~~~~~~
In file included from /home//src/highway/hwy/foreach_target.h:327,
                 from /home//src/highway/hwy/tests/mask_mem_test.cc:27:
/home//src/highway/hwy/tests/mask_mem_test.cc:377:19: note: ‘void hwy::N_SCALAR::TestAllStoreMaskBits()’ previously defined here
  377 | HWY_NOINLINE void TestAllStoreMaskBits() {
      |                   ^~~~~~~~~~~~~~~~~~~~
In file included from /home//src/highway/hwy/targets.cc:23,
                 from /home//src/highway/hwy/targets.h:345,
                 from /home//src/highway/hwy/highway.h:22,
                 from /home//src/highway/hwy/timer-inl.h:26,
                 from /home//src/highway/hwy/timer.cc:25,
                 from /home//src/highway/hwy/timer.h:67,
                 from /home//src/highway/hwy/nanobenchmark.h:52,
                 from /home//src/highway/hwy/tests/mask_mem_test.cc:23:
/home//src/highway/hwy/highway.h:226:38: error: ‘N_SSSE3’ has not been declared
  226 | #define HWY_CHOOSE_SSSE3(FUNC_NAME) &N_SSSE3::FUNC_NAME
  ```

Reply:
Unfortunately, error persists with this fix.

Reply:
hm, that's mystifying. Is that happening for all of the struct/class in this file?
TestStoreMaskBits is the only class, not that that should make a difference.

That class name is not repeated in other files. I'm not seeing anything different about this file compared to others.

Reply:
Thanks, indeed it is strange, there must be something else about this test that is different from all the others. Can you try on your system with the header only flag enabled ?

Reply:
I've currently got lots going on. Would anyone else be interested in having a look?